DTDL/replay_value_source.json

```diff
@@ -0,0 +1,50 @@
+{
+    "@context": "dtmi:dtdl:context;2",
+    "@id": "dtmi:io:tributech:source:replay;1",
+    "@type": "Interface",
+    "extends": "dtmi:io:tributech:source:base;1",
+    "displayName": "Replay Source",
+    "contents": [
+        {
+            "@type": "Property",
+            "name": "RowValueMetadataId",
+            "schema": "string",
+            "writable": true,
+            "displayName": "RowValueMetadataId",
+            "description": "The value MetadataId for the row json object"
+        },
+        {
+            "@type": "Property",
+            "name": "FileName",
+            "schema": "string",
+            "writable": true,
+            "displayName": "Frequency",
+            "description": "Line by line interval in ms"
+        },
+        {
+            "@type": "Property",
+            "name": "Frequency",
+            "schema": "double",
+            "writable": true,
+            "displayName": "Frequency",
+            "description": "Line by line interval in ms"
+        },
+        {
+            "@type": "Property",
+            "name": "RestartAutomatically",
+            "schema": "boolean",
+            "writable": true,
+            "displayName": "Frequency",
```

Comment:
Restart Automatically

Reply:
i think a better name for the property and display name would be even better. something like infinite replay